csharp-basic-topics/NonNullablePropertyMustContainANonNullValueWarning/Tests/NonNullablePropertyWarning.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NonNullablePropertyWarning;
+
+namespace Tests
+{
+    [TestClass]
+    public class NonNullablePropertyWarning
```

Comment:
The name of the class doesn't respect the naming convention, alongside the filename.

Reply:
Thanks, fixed.